velox/type/parser/TypeParser.ll

```diff
@@ -36,7 +36,6 @@ Z   [Z|z]
 WORD              ([[:alpha:][:alnum:]_]*)
 QUOTED_ID         (['"'][[:alnum:][:space:]_]*['"'])
 NUMBER            ([[:digit:]]+)
-ROW               (ROW|STRUCT)
```

Comment:
Looks like presto does not like the "struct" keyword.

Reply:
I think I followed the ANTLR grammar. We can always add this back later.

Reply:
At this least this does not seem to work for me:

```
presto:de> select cast(null as struct(bigint));
Query 20231220_202534_73200_nyxqh failed: line 1:8: Unknown type: struct(bigint)

presto:de> select cast(null as row(bigint));
 _col0                
-------
 NULL  
(1 row)

```